Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -538,6 +538,37 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.99,
 )
 
+class UpgradeWorkflow_TestOldDigi(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step:
+            # use existing DIGI-RAW file from old release
+            stepDict[stepName][k] = merge([{'--filein': 'das:/RelValTTbar_14TeV/CMSSW_11_0_0_pre13-110X_mcRun4_realistic_v2_2026D49noPU-v1/GEN-SIM-DIGI-RAW'}, stepDict[step][k]])
```

Comment:
should we use the 11.0.0 relval, `/RelValTTbar_14TeV/CMSSW_11_0_0-110X_mcRun4_realistic_v2_2026D49noPU-v2/GEN-SIM-DIGI-RAW` ?

Reply:
since there's no PU one (yet), I was just going to wait and update it later.

Reply:
makes sense